src/client/app/utils/api/ApiBackend.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import axios, {AxiosRequestConfig, AxiosResponse} from 'axios';
+import {getToken, hasToken} from '../token';
+
+/**
+ * Handles the actual sending and receiving of network requests.
+ *
+ * Should probably only be used by other api classes.
```

Comment:
Recommend removing "probably", we definitely don't want it used outside other API classes.